### DISCUSSION TITLE
How to know which memory particle is faulty in a memory?

### DISCUSSION THREAD

**Discussion by sstscrypto** [memtest86plus/memtest86plus#81] [Q&A]
example
DDR3 8GB 2R 16 chips
 If an error occurs, how do you know which chip is at fault?

**Comment by debrouxl** [comment]
The various Error Detection And Correction (EDAC) controllers built into many modern MBs can pinpoint the offending DIMM and the corresponding row / rank, as detailed at e.g. https://support.siliconmechanics.com/portal/en/kb/articles/how-to-diagnose-memory-errors-on-amd-x64-using-edac , but not necessarily the individual faulty chip, IIUC.

**Comment by sstscrypto** [comment]
> The various Error Detection And Correction (EDAC) controllers built into many modern MBs can pinpoint the offending DIMM and the corresponding row / rank, as detailed at e.g. https://support.siliconmechanics.com/portal/en/kb/articles/how-to-diagnose-memory-errors-on-amd-x64-using-edac , but not necessarily the individual faulty chip, IIUC.

thank you so much
